test/system/bootcode-dependencies.test.js

```diff
@@ -19,20 +19,25 @@ describe('bootcode dependencies', function () {
 
     it('should not change', function () {
         expect(currentDependencies).to.be.eql([
+            'ansi-styles',
             'array-uniq',
             'assert',
             'assertion-error',
             'atob',
             'backbone',
             'base64-js',
             'boolbase',
+            'browserify',
```

Comment:
How did this get in?

Reply:
This is not the actual browserify dev dependency. The empty module for ‘fs’ is under the browserify namespace. 

This is the namespace for builtin modules.

I have made sure that this is included only by the fs dependency and is actually empty.

Reply:
Here's a the more detailed version of the data

Without `os` and `fs` builtins.
```
6448819 bootcode.browser.js
6563213 bootcode.js
```

With `os` and `fs` builtins.
```
6451921 bootcode.browser.js
6566293 bootcode.js
```

Net size difference
```
3102 bootcode.browser.js
3080 bootcode.js
```

Given the size of the `os` shim is `973` characters and our bootcode is stringified buffer.